External/FEXCore/include/FEXCore/Debug/X86Tables.h

```diff
@@ -418,6 +436,9 @@ struct X86InstInfo {
     // We don't care if the opcode dispatcher differs
     return true;
   }
+  bool operator!=(const X86InstInfo *b) const {
```

Comment:
Looks like a good cleanup, just the buildbots don't like converting this pointer to a reference.